### PR TITLE
Daily dev digest template

### LINKED ISSUE #19: Create Daily Digest Email ERB file
- [x] Create daily_digest_email.html.erb and base structure off the original digest email file.
- [x] Modify so that presents the entire article instead of just a link.
      - Still needs to be tested
- [x] Styling (need mobile and a fixed width that works in a web browser email size)

### PR DESCRIPTION
**Pull request by grayson-palmer**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Mailer template for the new Daily Dev Digest email.

## Related Tickets & Documents
Resolves #19 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
N/A

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/mailers/digest_mailer/daily_dev_email.html.erb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+
+<h1 style="text-align: center;">
+  DEV Daily Digest
+</h1>
```
Comment (mel-rob) [inline app/views/mailers/digest_mailer/daily_dev_email.html.erb]: Can we add some tech-focused emojis in here to inject a little more fun? Thinking a row under the header?

Comment (mel-rob) [inline app/views/mailers/digest_mailer/daily_dev_email.html.erb:5]: Can we add some tech-focused emojis in here to inject a little more fun? Thinking a row under the header?

**Review by mel-rob — CHANGES_REQUESTED** [review]
Would like to inject a bit more fun in the template with the addition of some more emojis. Can we add a line of them under the header? Otherwise looks great, thank you!

**Review by mel-rob — APPROVED** [review]

File: app/views/mailers/digest_mailer/daily_dev_email.html.erb (hunk 1 above)
Comment (grayson-palmer) [reply app/views/mailers/digest_mailer/daily_dev_email.html.erb:5]: Added some emojis in to make it a bit more interesting

**Pull request merged**